src/generated/resources/assets/dungeonsdelight/lang/zh_cn.json

```
@@ -0,0 +1,402 @@
+{
+  "block.dungeonsdelight.cut_stained_scrap": "切制污秽残片",
+  "block.dungeonsdelight.cut_stained_scrap_slab": "切制污秽残片台阶",
+  "block.dungeonsdelight.cut_stained_scrap_stairs": "切制污秽残片楼梯",
+  "block.dungeonsdelight.dungeon_stove": "地牢炉灶",
+  "block.dungeonsdelight.embedded_eggs": "嵌入蛋",
+  "block.dungeonsdelight.glow_berry_gelatin_block": "发光浆果凝胶块",
+  "block.dungeonsdelight.guardian_angel_block": "守护天使",
+  "block.dungeonsdelight.heap_of_ancient_eggs": "远古蛋堆",
+  "block.dungeonsdelight.living_fire": "活焰",
+  "block.dungeonsdelight.monster_cake": "怪物蛋糕",
+  "block.dungeonsdelight.monster_pot": "怪物锅",
+  "block.dungeonsdelight.ossobusco_block": "红烩人小膝",
```

```suggestion
  "block.dungeonsdelight.ossobusco_block": "红烩人膝",
```
红烩**小牛**膝（Ossobusco）不是红烩牛**小膝** ；）
参见：[意大利烩牛膝](https://zh.wikipedia.org/zh-cn/%E6%84%8F%E5%A4%A7%E5%88%A9%E7%87%B4%E7%89%9B%E8%86%9D)

---

疑似为Osso-Buco 与 Osseous的组合词，或许可译为“红烩膝骨”